hypertrace-gradle-publish-maven-central-plugin/src/main/java/org/hypertrace/gradle/publishing/PublishMavenCentralPlugin.java

```diff
@@ -117,6 +117,9 @@ private void addJavaLibraryPublicationWhenApplied(PublicationContainer publicati
 
         // configure pom
         publication.pom(mavenPom -> {
+          // name
+          mavenPom.getName().set(project.getGroup().toString() + ":" + project.getName());
```

Comment:
Can this be set lazily? If the project group is changed later in the build script, would it pick it up? (that's basically just wrapping this in `mavenPom.getName().set(project.provider(() -> ...)`

Reply:
done.